README.md

```diff
@@ -138,18 +138,50 @@ ExecStart=/usr/bin/vip-manager --config=/etc/default/vip-manager.yml
 ```
 
 ## Configuration - Hetzner
-To use vip-manager with Hetzner Robot API you need a Credential file, set hosting_type to `hetzner` and your Floating-IP must be added on all Servers.
-The Floating-IP (VIP) will not be added or removed on the current Master node interface, Hetzner will route it to the current one.
 
-Set `hosting_type` to `hetzner` in `/etc/default/vip-manager.yml`
+To use vip-manager with Hetzner you need to configure the
+`/etc/hetzner` credentials file.
+
+Hetzner has two kinds of VIPs: the floating-IP and the failover-IP.
+
+For both kinds of VIPs you'll need to set up the failover-ip on all
+servers on the respective interface.
+
+vip-manager will not add or remove the VIP on the current master
+node interface, Hetzner will route it to the current one.
+
+#### FailoverIP
+
+[Hetzner failover-IP documentation](https://wiki.hetzner.de/index.php/Failover/en)
+[Hetzner Robot failover-IP API documentation](https://robot.your-server.de/doc/webservice/en.html#failover)
+
+* set `manager-type` to `hetzner` in `/etc/default/vip-manager.yml`
+* configure credentials in `/etc/hetzner`:
```

Comment:
Full agreement. We could then also get rid of the crazy credentials parsing [as](https://github.com/cybertec-postgresql/vip-manager/pull/58#discussion_r504061595) [you](https://github.com/cybertec-postgresql/vip-manager/pull/58#discussion_r504063022) [notice](https://github.com/cybertec-postgresql/vip-manager/blob/e9cd29e4defe8bed863090886a86a7b044f7972c/ipmanager/hetznerConfigurer.go#L78).

Reply:
Do you think we should change this first, change this PR and then merge? Or merge this PR (as long as you say that it works :smile: ) and afterwards change it for both "ip managers"?

Reply:
> Do you think we should change this first, change this PR and then merge? Or merge this PR (as long as you say that it works smile ) and afterwards change it for both "ip managers"?

To be honest, the clean and proper path would be to fix the technical debt in both "IP managers" first. On the other hand I dread having to keep on rebasing the "floating IP manager" on and on forever :-(

Also, having already poured considerable time into this I full understand the point of view of splitting out "other IP managers" out into their own repos and "plugins" via an API: it makes social organizational sense ("this is YOUR itch to scratch, not mine, so please YOU do the work and not offload it to me"). On the other hand: having one single lonely plugin is sad and needing to do the API rework and to *maintain* that is an additional burden. If you include the new, additional "IP manager" now, you can still drop it, if it turns out to suck, drain your energy and being there without anybody stepping up to fix/maintain it.

I leave the decision up to you. I appreciate each variant. I'd just like to ask for one favor please: whichever approach you choose please make it so that I don't have to maintain the "floating IP manager" as a fork of your project forever on :-/